src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/ByteArrayArgumentBindingProvider.cs

```diff
@@ -1,83 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-using System;
-using System.Reflection;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Host.Bindings;
-using Microsoft.Azure.WebJobs.Host.Storage.Queue;
-
-namespace Microsoft.Azure.WebJobs.Host.Queues.Bindings
-{
-    internal class ByteArrayArgumentBindingProvider : IQueueArgumentBindingProvider
-    {
-        private readonly IContextGetter<IMessageEnqueuedWatcher> _messageEnqueuedWatcherGetter;
-
-        public ByteArrayArgumentBindingProvider(IContextGetter<IMessageEnqueuedWatcher> messageEnqueuedWatcherGetter)
```

Comment:
It's  passed into the QueueBindingRules and flows through to the new AsyncCollector. 

---
In reply to: [62358535](https://github.com/Azure/azure-webjobs-sdk/pull/711#discussion_r62358535) [](ancestors = 62358535)